webhook/resources/backupbackingimage/validator.go

```
@@ -0,0 +1,54 @@
+	backupBackingImage, ok := newObj.(*longhorn.BackupBackingImage)
+	if !ok {
+		return werror.NewInvalidError(fmt.Sprintf("%v is not a *longhorn.BackupBackingImage", newObj), "")
+	}
+	backingImageName := backupBackingImage.Name
+
+	backingImage, err := bbi.ds.GetBackingImageRO(backingImageName)
+	if err != nil {
+		return werror.NewInvalidError(fmt.Sprintf("failed to get the backing image for backup", backingImage.Spec.SourceParameters, backingImage.Spec.SourceType), "")
+	}
+	if backingImage.Spec.DataEngine == longhorn.DataEngineTypeV2 {
+		return werror.NewInvalidError(fmt.Sprintf("v2 backing image is not supported for backup"), "")
```

golangci-lint (1.62.2)

S1039: unnecessary use of fmt.Sprintf

(gosimple)